### PR TITLE
PandiyanCool's First Goodie

### PR DESCRIPTION
**Pull request by PandiyanCool**
**What does your Instant Answer do?**
First goodie

**What problem does your Instant Answer solve (Why is it better than organic links)?**
First goodie

**What is the data source for your Instant Answer? (Provide a link if possible)**

**Why did you choose this data source?**

**Are there any other alternative (better) data sources?**

**What are some example queries that trigger this Instant Answer?**

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

**Which existing Instant Answers will this one supercede/overlap with?**

**Are you having any problems? Do you need our help with anything?**

**Where did you hear about DuckDuckHack? (For first time contributors)**

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
Please place an 'X' where appropriate.

```
[] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

**Comment by moollaza** [comment]
@PandiyanCool thanks a lot for contributing (we really appreciate it :smiley:) and congrats on making your first Goodie!

This LGTM :+1:

**Comment by moollaza** [comment]
@PandiyanCool we'll get this live shortly! Now that you've made your first Goodie, we really hope you'll continue to contribute and make more Instant Answer! Please checkout our [Ideas forum](https://duck.co/ideas) if you're unsure about what to build. There's also lots of Github Issues in our repos that need someone to tackle them!

**Comment by PandiyanCool** [comment]
thanks @moollaza for the update.

File: t/IsAwesome/PandiyanCool.t [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+use warnings;
+use Test::More;
+use DDG::Test::Goodie;
+
+zci answer_type => "is_awesome_pandiyan_cool";
+zci is_cached   => 1;
+
+ddg_goodie_test(
+    [qw(
+        DDG::Goodie::IsAwesome::pandiyancool
```
Comment (moollaza) [inline t/IsAwesome/PandiyanCool.t]: Just realized this needs to be PandiyanCool (case sensitive!) will fix

Comment (PandiyanCool) [inline t/IsAwesome/PandiyanCool.t]: @moollaza 

so this should changed as follows?

DDG::Goodie::IsAwesome::PandiyanCool

Comment (moollaza) [inline t/IsAwesome/PandiyanCool.t]: Yup, I already did it :)

Comment (PandiyanCool) [inline t/IsAwesome/PandiyanCool.t]: Great!! Thank you!!

**Comment by moollaza** [comment]
@PandiyanCool congrats! It's live: https://duckduckgo.com/?q=duckduckhack+pandiyancool&ia=answer

**Comment by PandiyanCool** [comment]
thanks @moollaza

**Pull request merged**